client/my-sites/checkout/checkout/domain-details-form.jsx

```diff
@@ -195,19 +206,33 @@ export class DomainDetailsForm extends PureComponent {
 	};
 
 	handleChangeEvent = event => {
-		// Resets the state field every time the user selects a different country
+		this.shouldFocusAddress = false;
```

Comment:
Do we need this? We only want to suppress the address focus for the initial render, so we could just leave it true, couldn't we?

If we do want it to be more like a consumable flag, I'd prefer to turn it off in callback where we set focus if we can. This says "Every time any input changes, don't focus on the address" which is a bit weird.

Would it be clearer maybe if we reversed the logic and called it something like `skipFocusForInitialRender: true` and then set it to false in 'country-code' branch?

Reply:
>> Do we need this?
Now that I look at it, no we don't.

>> We only want to suppress the address focus for the initial render, so we could just leave it true, couldn't we?
Yes.

>> Would it be clearer maybe if we reversed the logic and called it something like skipFocusForInitialRender: true and then set it to false in 'country-code' branch?

If we got rid of the flag reset, I think it would be clearer yes! Good one. 👍